lib_test/test_fs_link_chunck.ml

```diff
@@ -0,0 +1,49 @@
+(*
+ * Copyright (c) 2013 Thomas Gazagnaire <thomas@gazagnaire.org>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+open Test_common
+
+let test_db = "test-db-chunck"
+
+let polling =
+  try float_of_string (Sys.getenv "IRMIN_FS_POLLING")
+  with Not_found | Failure _ -> 0.01
+
+let init () =
+  Irmin_unix.install_dir_polling_listener polling;
+  if Sys.file_exists test_db then begin
+    let cmd = Printf.sprintf "rm -rf %s" test_db in
+    let _ = Sys.command cmd in ()
+  end;
+  Lwt.return_unit
+
+let clean () =
+  Irmin_unix.uninstall_dir_polling_listener ();
+  Lwt.return_unit
+    
+let config = Irmin_fs.config ~root:test_db ()
+let config = Irmin_chunck.config ~conf:config ~size:4096 ()
+    
+let suite k =
+  {
+    name   = "LINK FS - CHUNCK" ^ string_of_contents k;
+    kind   = `Link_FS_Chunck;
+    cont   = k;
+    init   = init;
+    clean  = clean;
+    store  = link_fs_chunck_store k;
+    config = config;
+  }
```

Comment:
All the new tests will benefit avoiding code duplication and be more functorised. I don't really want to copy/past more code if I decide to test chunk over chunck over http over memory. I just want to apply a functor (see how it''s done for the http backend tests).